app/routes/basic.js

```diff
@@ -31,9 +31,6 @@ export default Ember.Route.extend({
     return this.get('auth.currentUser');
   },
 
-  // on pro, we need to auth on every route
-  needsAuth: Ember.computed.alias('features.proVersion'),
```

Comment:
Good catch! iirc I was having some test issues that I was trying to work around. I had made a note to revisit it, but simply forgot.